### ISSUE TITLE
[0.9.0 staging-1646] Shaders: Invisible claims put you down with errors

### ISSUE THREAD

**Issue by beastyn** [StrangeLoopGames/EcoIssues#17031]
1. Take a claim stake 
Error is spamming in the log till your FPS is at the bottom line.
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/4889166/Player.log)

2. Try to claim something
No visible deeds.
![bandicam 2020-07-08 12-42-04-031](https://user-images.githubusercontent.com/27898520/86891829-e004ac80-c118-11ea-8545-e90b87dca7b2.jpg)

**Comment by SlayksWood** [comment]
recheck #12361

**Comment by beastyn** [comment]
@redwyre I have checked this issue locally, and I did not have it in Unity. But we have it in the last builds.
Yesterday we had a problem with client build  in 1647. Error was like that
Failed to get socket connection from UnityShaderCompiler.exe shader compiler! C:/Program Files/Unity/Hub/Editor/2019.4.1f1/Editor/Data/Tools/UnityShaderCompiler.exe 
(Filename: C:\buildslave\unity\build\Tools/UnityShaderCompiler/ShaderCompilerClient.cpp Line: 339)
